nginx/nginx.go

```diff
@@ -402,12 +402,14 @@ func createUpstreamEntries(entries controller.IngressEntries) []*upstream {
 	idToUpstream := make(map[string]*upstream)
 
 	for _, ingressEntry := range entries {
-		upstream := &upstream{
-			ID:             upstreamID(ingressEntry),
-			Server:         fmt.Sprintf("%s:%d", ingressEntry.ServiceAddress, ingressEntry.ServicePort),
-			MaxConnections: ingressEntry.BackendMaxConnections,
+		if validUpstreamEntry(ingressEntry) {
```

Comment:
I think we would want some sort of log for when we get an invalid ingress. We could maybe aggregate the invalid ingresses and then log them as a warn?

Reply:
Its nice Idea to log, I am not sure aggregating though, as we do not expect loads of them.